OpenNefia.Content/UI/Element/UiWrapText.cs

```diff
@@ -0,0 +1,105 @@
+﻿using OpenNefia.Core.Maths;
+using OpenNefia.Core.Rendering;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.UI.Element
+{
+    public class UiWrapText : UiText
+    {
+        private int MaxWidth;
+        public UiWrapText(int maxWidth, FontSpec font, string text = "") : base(font, text)
+        {
+            MaxWidth = maxWidth;
+            Text = text;
+        }
+
+
```

Comment:
Nitpick: extra indentation?

Reply:
do you mean that there's two newlines instead of one?

Reply:
(in that case done in 76f6762)